### PR TITLE
fix: [ion-177] selection adjustment

### PR DESCRIPTION
**Pull request by MarcosOliveir4**

### CODE REVIEW

File: projects/ion/src/lib/tab-group/tab-group.component.ts [hunk 1]
```diff
@@ -1,4 +1,11 @@
-import { Component, EventEmitter, Input, Output, OnInit } from '@angular/core';
+import {
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  OnInit,
+  ChangeDetectorRef,
```
Comment (iurynogueira) [inline projects/ion/src/lib/tab-group/tab-group.component.ts:7]: is defined but never used.

File: projects/ion/src/lib/tab/tab.component.ts [hunk 2]
```diff
@@ -33,6 +33,8 @@ export class TabComponent {
   @Input() badge?: BadgeOptions;
 
   select() {
-    this.selected = !this.selected;
+    if (!this.selected) {
+      this.selected = !this.selected;
+    }
```
Comment (iurynogueira) [inline projects/ion/src/lib/tab/tab.component.ts:38]: I believe that you can do this.selected = true.
```suggestion
      this.selected = true;
```

File: projects/ion/src/lib/tab-group/tab-group.component.ts [hunk 3]
```diff
@@ -47,9 +54,11 @@ export class TabGroupComponent implements OnInit {
   }
 
   public selectTab(tabSelected: TabInGroup) {
-    this.clearTabs();
-    tabSelected.selected = true;
-    this.selected.emit(tabSelected);
+    if (!tabSelected.selected) {
+      this.clearTabs();
+      tabSelected.selected = true;
```
Comment (iurynogueira) [inline projects/ion/src/lib/tab-group/tab-group.component.ts:59]:
```suggestion
      tabSelected.select();
```

Call method, this way if there is a change in logic the behavior will be replicated here.

**Review by iurynogueira — CHANGES_REQUESTED** [review]

**Review by AlyssonMascarenhas — CHANGES_REQUESTED** [review]
Github Actions / test-prettier-lint:
'ChangeDetectorRef' is defined but never used.

**Review by Deeved — APPROVED** [review]

**Review by allanplobo — APPROVED** [review]

**Review by iurynogueira — APPROVED** [review]
nice work marquinhos!

festa

**Review by mizek1 — APPROVED** [review]

**Review by AlyssonMascarenhas — APPROVED** [review]

**Pull request merged**